### PR TITLE
migrate 1.8.9 to new intermediaries

### PR DESCRIPTION
**Pull request by BluCobalt**
if this works i will release my scripts and port the change to the other versions/branches

### CODE REVIEW

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping [hunk 1]
```diff
@@ -1,361 +1,361 @@
-CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
+CLASS net/minecraft/class_2403 com/mojang/blaze3d/platform/GlStateManager
 	COMMENT Allows calling the static methods in {@code GL11} without actually touching it
-	FIELD field_4311 ALPHA_TEST Lnet/minecraft/class_1013$class_1014;
+	FIELD field_10708 ALPHA_TEST Lnet/minecraft/class_1013$class_1014;
```
Comment (arthurbambou) [inline mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:3]: Ugh, it didn't remap the field type.

File: mappings/com/mojang/blaze3d/platform/GlStateManager.mapping [hunk 2]
```diff
@@ -1,361 +1,361 @@
...
 		ARG 1 dstFactorRGB
 		ARG 2 srcFactorAlpha
 		ARG 3 dstFactorAlpha
-	METHOD method_3412 getFloat (ILjava/nio/FloatBuffer;)V
+	METHOD method_9806 getFloat (ILjava/nio/FloatBuffer;)V
 		COMMENT Gets a float for a float buffer
 		ARG 0 mode
 		ARG 1 buffer
-	METHOD method_3413 enableTexCoord (Lnet/minecraft/class_1013$class_1028;)V
+	METHOD method_9807 enableTexCoord (Lnet/minecraft/class_1013$class_1028;)V
```
Comment (arthurbambou) [inline mappings/com/mojang/blaze3d/platform/GlStateManager.mapping:114]: Same with method parameters type.

**Review by arthurbambou — CHANGES_REQUESTED** [review]

**Comment by arthurbambou** [comment]
But it built without errors so I'll give them a try

**Comment by BluCobalt** [comment]
I think that's only on the com/Mojang classes because everything else looks changed properly

**Review by arthurbambou — APPROVED** [review]

**Review by arthurbambou — CHANGES_REQUESTED** [review]
I've checked again and the only issue remaining is inner class name not being remapped.

**Review by arthurbambou — APPROVED** [review]

**Review by Grayray75 — APPROVED** [review]

**Pull request merged**